src/main/java/com/github/timm/cucumber/generate/CucumberITGenerator.java

```diff
@@ -193,9 +193,18 @@ private String createFormatStrings() {
 
         for (int i = 0; i < formatStrs.length; i++) {
             final String formatStr = formatStrs[i].trim();
+            String extension = formatStr;
+            
+            //Supported Report Types - https://cucumber.io/docs/reference#reports 
+            if(extension.equals("rerun")){
+                extension = "txt";
+            } else if(extension.equals("junit")){
+                extension = "xml";
+            }
+            
             sb.append(String.format("\"%s:%s/%s.%s\"", formatStr,
                     config.getCucumberOutputDir()
-                    .replace('\\', '/'), fileCounter, formatStr));
+                    .replace('\\', '/'), outputFileName, extension));
```

Comment:
using the outputFileName instead of fileCounter since I think it makes sense to make the output files match the associated IT Generated file.

I would think this just makes it easier to determine the associated IT file.